dev/io.openliberty.microprofile.openapi.4.0.internal.services/.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -140,13 +140,13 @@ cleanup_settings_version=2
 eclipse.preferences.version=1
 editor_save_participant_org.eclipse.jdt.ui.postsavelistener.cleanup=true
 formatter_profile=_OL Standard
-formatter_settings_version=21
+formatter_settings_version=23
 org.eclipse.jdt.ui.ignorelowercasenames=true
 org.eclipse.jdt.ui.importorder=java;javax;org;com;
-org.eclipse.jdt.ui.javadoc=false
+org.eclipse.jdt.ui.javadoc=true
```

Comment:
Was this switch over from `false` to `true` intended?

Reply:
Yes, despite its name, this preference controls whether all types of comment stub are created for new files, classes, methods etc.

I particularly want it on to create the copyright header in new files.